www/js/con-student.js

```diff
@@ -496,17 +496,19 @@ define([
 			/*
 			 Set color and enable/disable
 			 */
-			array.forEach(this._model.student.getStatusDirectives(nodeid), function (directive) {
-				var w = registry.byId(this.controlMap[directive.id]);
-				w.set(directive.attribute, directive.value);
-				// The actual values should be in the model itself, not in status directives.
-				if (directive.attribute == "value") {
-					this.logging.clientLog("error", {
-						message: "Values should not be set in status directives",
-						functionTag: 'initialControlSettings'
-					});
-				}
-			}, this);
+			if(this._PM.mode !== "EDITOR" && this._PM.mode !== "TEST"){
```

Comment:
sorry this is not to be done like this. Create activity-parameter and set this as a field and check that field whether it is tru or false.